File: C_Programming/Competitive_Programming/Assignment12/program2.c
//Write a program which accept number from user check whether it contains 0 in it or not

// Input : 2395
// Output : There is not zero

// Input : -1018
// Output : It contains Zero//              

#include<stdio.h>

#define TRUE 1
#define FALSE 0

typedef int BOOL;

BOOL ChkZero(int iNo)
{
    int iDigit = 0;

    while(iNo > 0)
    {
        iDigit = iNo % 10;
        if(iDigit == 0)
        {
            return TRUE;
        }
        iNo = iNo / 10;
    }

    return FALSE;
}

int main()
{
    int iValue = 0;
    BOOL bRet = FALSE;

    printf("Enter number : ");
    scanf("%d",&iValue);

    bRet = ChkZero(iValue);

    if(bRet == 1)
    {
        printf("It contains Zero");
    }
    else
    {
        printf("There is no Zero");
    }

    return 0;
}

File: Java_Programming/Competitive_Programming/Assignment17/program2.java
// Write a program to check whether a number is a palindrome or not  

class program2
{
    public static void main(String A[] ) 
    {
        logic obj = new logic();
        obj.CheckPalindrome(121);
    }
}
class logic 
{
    void CheckPalindrome(int num)
    {
        int iDigit = 0;
        int rev = 0;
        int no = num;

        while (num > 0) 
        { 
            iDigit = num % 10;
            rev = (rev * 10) + iDigit;
            num = num / 10;
        }
     
        if(no == rev)
        {
            System.out.println("Number is Palindrome");
        }
        else
        {
            System.out.println("Number is not Palindrome");
        }

    }
}